src/arizona_tpl_parse.erl

```diff
@@ -24,7 +24,9 @@ Template parser.
 -moduledoc #{author => "William Fank Thomé <willilamthome@hotmail.com>"}.
 
 %% API functions.
--export([parse_exprs/1, parse_exprs/2]).
+-export([parse_exprs/1]).
+-ignore_xref([parse_exprs/1]).
```

Comment:
If the call of `parse_exprs/1` from [this line](https://github.com/arizona-framework/arizona/blob/7d4afb5a6ae13d090bda40110b5cbba1cc799c98/src/arizona_tpl_parse.erl#L331) of the code be changed to
```erlang
parse_exprs(Tokens, #{})
```
..then `parse_exprs/1` can be safely removed.

Reply:
Will do...

Reply:
a9740cc